example-files/notebooks/mango-variants-viz.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
can you rename to something more informative?

Reply:
I named it as currently  because it is about variants visualization, happy to change it to something else you might suggest.